### PR TITLE
Add note about missing web support in profiling.md

### PR DESCRIPTION
**Pull request by edhom**
Adds a note that tracing performance in integration tests is not supported on web.

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]
PTAL @atsansone

File: src/cookbook/testing/integration/profiling.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ In this recipe, learn how to write a test that records a performance
 timeline while performing a specific task and saves a summary of the
 results to a local file.
 
+{{site.alert.note}}
+  Note that recording performance timelines is not supported on web.
+  For performance profiling on web, see
+  [Debugging performance for web apps][]
```
Comment (sfshaza2) [inline src/cookbook/testing/integration/profiling.md:27]: You don't need to say "Note" as the note macro provides that.

```suggestion
  Recording performance timelines isn't supported on web.
  For performance profiling on web, see
  [Debugging performance for web apps][]
```

**Review by sfshaza2 — APPROVED** [review]
lgtm with one tweak

**Comment by domesticmouse** [comment]
@sfshaza2 is this now good to land?

**Comment by sfshaza2** [comment]
Yes! lgtm

**Pull request merged**